recipes/mbutil/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "mbutil" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 78fca26f2497c4e20592792e2fe4abd5fd51e58a03a023a42a54274b7c13c419
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - mbutil
+
+about:
+  home: https://github.com/mapbox/mbutil
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: An importer and exporter for MBTiles
+  dev_url: https://github.com/mapbox/mbutil
+
+extra:
+  recipe-maintainers:
+    - rluria14
```

Comment:
Shall you add the other maintainers?

Reply:
```suggestion
    - rluria14
    - oblute
    - benhuff
```

Reply:
changes have been made and maintainers added.